### PR TITLE
Update get-machineactions-collection.md

### PR DESCRIPTION
**Pull request by msbemba**
updated OData's $filter query

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5899

File: microsoft-365/security/defender-endpoint/get-machineactions-collection.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ Retrieves a collection of [Machine Actions](machineaction.md).
 
 Supports [OData V4 queries](https://www.odata.org/documentation/).
 
-The OData's `$filter` query is supported on: `status`, `machineId`, `type`, `requestor` and `creationDateTimeUtc` properties.
+The OData's `$filter` query is supported on: `id`, `status`, `machineId`, `type`, `requestor` and `creationDateTimeUtc` properties.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-machineactions-collection.md]:
```suggestion
The OData's `$filter` query is supported on: `id`, `status`, `machineId`, `type`, `requestor`, and `creationDateTimeUtc` properties.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**